### PR TITLE
Add service mark_dhcp_packet to mux container

### PR DESCRIPTION
**Pull request by ghost**
- add a new service "mark_dhcp_packet" to mux container
- apply packet marks on a per-interface basis in ebtables
- write packet marks to "DHCP_PACKET_MARK" table in state_db

Example:
INPUT chain in ebtables:
ebtables

DHCP_PACKET_MARK in state_db:
state_db

### CODE REVIEW

File: files/scripts/mark_dhcp_packet.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        mux_cables = self.config_db.get_table('MUX_CABLE')
+        mux_intfs = [intf for intf in mux_cables if intf.startswith('Ethernet')]
+
+        return mux_intfs
+
+    def generate_mark_for_intf(self, intf):
+        '''
+        type: string, format: hexadecimal
+        '''
+        intf_index_str = intf[8:]
```
Comment (prsunny) [inline files/scripts/mark_dhcp_packet.py:69]: Please specify an example here. Another issue is, if we ever change the name to say etp1, then this logic fails. So please revise the logic to uniquely identify the interface index

File: files/scripts/mark_dhcp_packet.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        intf_mark = "0x67" + intf_index_str.zfill(3)
+
+        return intf_mark
+
+    def run_command(self, cmd):
+        subprocess.call(cmd, shell=True)
+        log.log_info("run command: {}".format(cmd))
+
+    def clear_dhcp_packet_marks(self):
+        self.run_command("sudo ebtables -F INPUT")
```
Comment (prsunny) [inline files/scripts/mark_dhcp_packet.py:79]: This needs to be documented.

**Comment by trzhang-msft** [comment]
> This commit could not be cleanly cherry-picked to 202012. Please submit another PR.

I don't see mux docker template file in 202012. Please solve that problem first. "mark_dhcp_packet" is a service inside mux container.

**Comment by trzhang-msft** [comment]
> This commit could not be cleanly cherry-picked to 202012. Please submit another PR.

Please check if this pr now can be cherry-picked. If not, I will submit another.

**Comment by qiluo-msft** [comment]
I cherry-picked again and it failed.

**Pull request merged**